src/pages/index.js

```diff
@@ -1,93 +1,107 @@
 /** @jsx jsx */
-import { jsx } from '@emotion/core'
+import { jsx } from "@emotion/core"
 // eslint-disable-next-line
 import { graphql } from "gatsby"
 
 import NonStretchedImage from "../components/nonStretchedImage"
 import Layout from "../templates/layout"
 import SEO from "../components/seo"
 import Helmet from "react-helmet"
-import { Box, Flex, H1, H3, H4, P, ReactIcon } from 'bricks'
-import PlainLink from '../components/link'
+import { Box, Flex, H1, H3, H4, P, ReactIcon } from "bricks"
+import PlainLink from "../components/link"
 
-const structuredData = `{
-  "@context" : "http://schema.org",
-  "@type" : "LocalBusiness",
-  "name" : "Codebrahma",
-  "image" : "https://codebrahma.com/static/690ea05ce05812b0bc0798a71a210f4b/d042c/rippling_screenshot.png",
-  "telephone" : "+1 484 506 0634",
-  "email" : "hello@codebrahma.com",
-  "address" : {
-    "@type" : "PostalAddress",
-    "streetAddress" : "156, 2nd Street",
-    "addressLocality" : "San Francisco,",
-    "addressRegion" : "CA",
-    "postalCode" : "94105"
-  }
-}`
-
-const IndexPage = ({data}) => (
-<Layout>
-
-  <Helmet>
-    <script type="application/ld+json">
-      {structuredData}
-    </script>    
-  </Helmet>
-  <SEO 
-    title="Best ReactJS ReactNative Consultancy in Bay Area" 
-    keywords={[`react`,`react native`, `reactJS`, `consultancy`, `bay area`]} 
-    description=''
-  />
-  <Box mt='6' mb='6'>
-    <H1 lineHeight='42px'>We build your React &amp;&nbsp;React Native frontend</H1>
-    <P>We take your design files &amp; api docs. And give you a fast mobile and browser app.
-      You focus on your backend business&nbsp;logic!</P>
-  </Box>
-  <div>
-    <H3>
-      Our Featured Work
-    </H3>
-    <Flex flexDirection='column'>
-      <PlainLink to='/rippling/'>
-        <H4>
-          Rippling
-        </H4>
-      </PlainLink>
-      <NonStretchedImage fluid={data.rippling_screenshot.childImageSharp.fluid} />
-      <P>Codebrahma built the complete browser and mobile app for this fast growing
-        Parker Conrad startup, currently valued at $250 million</P>
-    </Flex>
-  </div>
-  <div>
-    <H3>
-      React & ReactNative
-    </H3>
-    <Flex flexDirection='column'>
-      <Box alignSelf={['center', 'flex-start']} width={1}>
-        <ReactIcon width="200" height="200" />
+const IndexPage = ({ data }) => {
+  
+  const structuredData = `{
+    "@context" : "http://schema.org",
+    "@type" : "LocalBusiness",
+    "name" : "Codebrahma",
+    "image" : "https://codebrahma.com${data.rippling_screenshot.publicURL}",
```

Comment:
@sanandnarayan do you think it is right to place rippling screenshot for codebrahma website?